src/main/java/io/github/soc/directories/Util.java

```diff
@@ -114,10 +116,11 @@ static String linuxExecutableDir(String homeDir, String dataDir) {
       buf.append("\\\")\n");
     }
 
-    return runCommands(guidsLength,
+    return runCommands(guidsLength, StandardCharsets.UTF_8,
```

Comment:
```suggestion
    return runCommands(guidsLength, Charset.forName("UTF-8"),
```
(`StandardCharsets` was only added in Java 7.)